src/GlobalX.ChatBots.WebexTeams/Services/HttpClientProxy.cs

```diff
@@ -44,7 +46,14 @@ public async Task<string> PostAsync(string path, string body = null)
                 RequestUri = new Uri($"{_settings.WebexTeamsApiUrl}{path}"),
                 Content = body != null ? new StringContent(body, Encoding.UTF8, "application/json") : null
             };
+
             var response = await _httpClient.SendAsync(request).ConfigureAwait(false);
+
+            if (response.StatusCode == HttpStatusCode.BadRequest)
```

Comment:
Is a bad request always going to indicate invalid parent, or is there something else about the response that will tell us it's an invalid parent? Want to avoid weird misleading error messages

Reply:
Considering we're formulating the request the same way every time, I can't think of any way it would be a bad request *other* than this.